### PR TITLE
Add new folder structure

### PR DESCRIPTION
**Pull request by JoeZiminski**
Update folder structure to new preferred heirarchy as discussed in #25 

e.g.

```
└── project_name/
    └── rawdata/
        ├── sub-001/
        │   └── ses-001/
        │       ├── ephys/
        │       └── behav/
        │   └── histology/
        └── sub-002/
            └── ses-001/
                ├── behav/
                └── imaging/
            └── ses-002/
                └── behav/
            └── histology/
```

### CODE REVIEW

**Comment by JoeZiminski** [comment]
TODO: remove TODOs

File: datashuttle/datashuttle.py [hunk 1]
```diff
@@ -59,6 +59,9 @@ def __init__(self, project_name: str):
         self.cfg: Any = None
         self._ssh_key_path: Any = None
         self._ses_dirs: Any = None
+        self._top_level_dir_name = (
+            "rawdata"  # TODO: move to configs, make changable?
```
Comment (JoeZiminski) [inline datashuttle/datashuttle.py:63]: #33

**Comment by JoeZiminski** [comment]
That was scary, used force push --all and pushed an old branch that overwrote a load of commits. Luckily got back by using git reset --hard <most recent commit hash>. Note to self - never use force --all!

**Comment by adamltyson** [comment]
> Note to self - never use force --all!

This is why I just never force push!

**Comment by niksirbi** [comment]
When initialising a new project from the command line, I get the following:
```bash
datashuttle test-project make_config_file --ssh_to_remote False --remote_path_local /media/ceph/neuroinformatics/sirmpilatzen/test-project --use_ephys True --use_behav True --use_imaging True --use_histology True /home/nsirmpilatze/Data/test-project
/home/nsirmpilatze/Code/NIU/datashuttle/datashuttle/datashuttle.py:384: UserWarning: Configuration file has not been initialized. Use make_config_file() to setup before continuing.
  warnings.warn(
[remote_test-project_local]
type = local
Configuration file has been saved and options loaded into datashuttle.
```
I don't know why the warning appears here, since I am just now initialising the project, and as the subsequent output shows, it is indeed being initialised.

**Comment by niksirbi** [comment]
When running `datashuttle test-project upload_project_dir_or_file rawdata/sub-03`, I got:
```bash
<3>ERROR : Local file system at /home/nsirmpilatze/Data/test-project/rawdata/rawdata/sub-03: error reading source root directory: directory not found
<3>ERROR : Attempt 1/3 failed with 1 errors and: directory not found
<3>ERROR : Local file system at /home/nsirmpilatze/Data/test-project/rawdata/rawdata/sub-03: error reading source root directory: directory not found
<3>ERROR : Attempt 2/3 failed with 1 errors and: directory not found
<3>ERROR : Local file system at /home/nsirmpilatze/Data/test-project/rawdata/rawdata/sub-03: error reading source root directory: directory not found
<3>ERROR : Attempt 3/3 failed with 1 errors and: directory not found
Failed to copy: directory not found
```
It looks as if the path is not read relative to the project directory, but instead relative to the `rawdata` folder. Is that behaviour desired? If yes, docstrings and cli help message should change accordingly.

**Review by niksirbi — APPROVED** [review]

**Pull request merged**